tutorials/ndvi_hotspots.ipynb

```diff
@@ -0,0 +1,1063 @@
+   "source": [
+    "### Dask Cluster"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "https://pcc-staging.westeurope.cloudapp.azure.com/compute/services/dask-gateway/clusters/staging.a8086ff7049d497dbdf40c6b586e1ca6/status\n"
+     ]
+    }
+   ],
+   "source": [
+    "cluster = GatewayCluster()  # Creates the Dask Scheduler. Might take a minute.\n",
+    "\n",
+    "client = cluster.get_client()\n",
+    "\n",
+    "cluster.adapt(minimum=4, maximum=24)\n",
+    "print(cluster.dashboard_link)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['2016-01-01/2016-12-31',\n",
+       " '2017-01-01/2017-12-31',\n",
+       " '2018-01-01/2018-12-31',\n",
+       " '2019-01-01/2019-12-31',\n",
+       " '2020-01-01/2020-12-31']"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import datetime as dt\n",
+    "\n",
+    "datetimes = [f'{dt.date.min.replace(year=y)}/{dt.date.max.replace(year=y)}' for y in range(2016, 2021)]\n",
+    "\n",
+    "datetimes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The region of interest is a small region in Amazon rainforest, located in State of Mato Grosso and State of Amazonas, Brazil."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[-60.780782, -8.91121, -60.725047, -8.873188]"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "y1, x1 = -8.911210, -60.780782\n",
+    "y2, x2 = -8.873188, -60.725047\n",
+    "\n",
+    "bbox = [x1, y1, x2, y2]\n",
+    "bbox"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In order to calculate NDVI accurately, let's choose a single scene and find out its least cloudy image for each year."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def search(bbox, datetime, scene_id=None):\n",
+    "    # find least cloudy scene\n",
+    "    catalog = pystac_client.Client.open('https://planetarycomputer.microsoft.com/api/stac/v1')\n",
+    "    search = catalog.search(\n",
+    "        collections=['sentinel-2-l2a'], bbox=bbox, datetime=datetime,    \n",
+    "    )    \n",
+    "    if scene_id is not None:\n",
+    "        items = [item for item in search.items() if scene_id in item.id]\n",
+    "    else:\n",
+    "        items = search.items()\n",
+    "\n",
```

Comment:
`catalog.search` can take a  list of `ids` to find. Might simplify this by just taking `ids` in your `search` function and passing it along.

I also worry a bit that this searching / filtering is distracting from the point of the notebook: hotspot analysis. What do you think about just hardcoding the IDs we use in the notebook? Then we can say "we found these least cloudy scenes by searching with the STAC API and filtering the results. See the STAC API notebook for more.

And link to https://planetarycomputer.microsoft.com/docs/quickstarts/reading-stac/. I think you can do relative links like `../quickstarts/reading-stac.ipynb`.

Reply:
So something like

```python
ids = [
    "S2A_MSIL2A_20160627T142042_N0212_R010_T20LQR_20210211T234456.SAFE",
    "S2B_MSIL2A_20170727T142039_N0212_R010_T20LQR_20210210T153028.SAFE",
    "S2B_MSIL2A_20180712T142039_N0212_R010_T20LQR_20201011T150557.SAFE",
    "S2B_MSIL2A_20190617T142049_N0212_R010_T20LQR_20201006T032921.SAFE",
    "S2A_MSIL2A_20200616T141741_N0212_R010_T20LQR_20200822T232052.SAFE",
]
items = catalog.search(ids=ids)
```

and then stackstac.stack those (untested)